docs/BR.md

```diff
@@ -1754,10 +1754,6 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
 
 The CA SHALL meet the technical requirements set forth in [Section 2.2 - Publication of Information](#22-publication-of-information), [Section 6.1.5 - Key Sizes](#615-key-sizes), and [Section 6.1.6 - Public Key Parameters Generation and Quality Checking](#616-public-key-parameters-generation-and-quality-checking).
 
-TODO(sleevi): FIXME
-
-CAs SHALL generate non-sequential Certificate serial numbers greater than zero (0) containing at least 64 bits of output from a CSPRNG.
```

Comment:
The commit comment does not accurately and fully describe the associated changes.  That's what I was pointing out.

Reply:
Thanks for clarifying. I do believe the comment accurately describes what was changed - a TODO of stale text was removed, because it was addressed. I think you may just be confused about what the TODO was expressing, which was more intended for my benefit/reminder of the things to fix (and this was fixed)